src/main/java/org/cascadebot/cascadebot/utils/ColorUtils.java

```diff
@@ -0,0 +1,93 @@
+    private static final Pattern RGB_COLOR = Pattern.compile("(\\d{1,3}),(\\d{1,3}),(\\d{1,3})");
+    private static final Pattern BINARY_COLOR = Pattern.compile("([0-1]+)");
+
+    public String getHex(int r, int g, int b) {
+        return String.format("%02x%02x%02x", r, g, b);
+    }
+
+    public MessageEmbed getColor(String text, CommandContext context) {
+
+        Color color = null;
+        Matcher matcher;
+
+        try {
+            color = (Color) Color.class.getField(text.toUpperCase()).get(null);
+        } catch (IllegalArgumentException | IllegalAccessException | NoSuchFieldException | SecurityException e) {
+            //RGB_COLOR Values
+            if ((matcher = RGB_COLOR.matcher(text)).find()) {
+                try {
+                    color = new Color(Integer.parseInt(matcher.group(1)), Integer.parseInt(matcher.group(2)),
+                            Integer.parseInt(matcher.group(3)));
+                } catch (IllegalArgumentException e1) {
+                    context.getTypedMessaging().replyDanger(context.i18n("commands.color.valid_rgb"));
+                    return null;
+                }
+                //Hex
+            } else if ((matcher = HEX_COLOR.matcher(text)).matches()) {
+                try {
+                    color = Color.decode(matcher.group());
+                } catch (NumberFormatException e1) {
+                    context.getTypedMessaging().replyDanger(context.i18n("commands.color.valid_hex_code"));
+                    return null;
+                }
+                //Decimal
+            } else if ((matcher = DECIMAL_COLOR.matcher(text)).matches()) {
+                try {
+                    color = Color.decode(matcher.group());
+                } catch (NumberFormatException e1) {
+                    context.getTypedMessaging().replyDanger(context.i18n("commands.color.valid_decimal"));
+                    return null;
+                }
+                //Binary
+            } else if ((matcher = BINARY_COLOR.matcher(text)).matches()) {
+                try {
+                    color = Color.decode(String.valueOf(Integer.parseUnsignedInt(matcher.group(), 2)));
+                } catch (NumberFormatException e1) {
+                    context.getTypedMessaging().replyDanger(context.i18n("commands.color.valid_binary"));
+                }
+            }
+        }
+
+        if (color == null) {
+            context.getTypedMessaging().replyDanger(context.i18n("commands.color.color_not_recognised"));
+            return null;
+        }
+
+        String rgbValues = color.getRed() + "," + color.getGreen() + "," + color.getBlue();
+        String hex = getHex(color.getRed(), color.getGreen(), color.getBlue());
+        int decimalColor = Integer.parseUnsignedInt(hex, 16);
+
+        EmbedBuilder builder = MessagingObjects.getMessageTypeEmbedBuilder(MessageType.INFO, context.getUser());
+        builder.setTitle(context.i18n("commands.color.embed_title", hex));
```

Comment:
You need to move the lang strings now from commands to utils

Reply:
Got it